pkgs/applications/science/machine-learning/starspace/default.nix

```
@@ -0,0 +1,29 @@
+{lib, stdenv, fetchFromGitHub, boost165}:
+
+stdenv.mkDerivation rec {
+  pname = "starspace";
+  version = "20210117-${lib.strings.substring 0 7 rev}";
```

As per CONTRIBUTING.md:

```suggestion
  version = "unstable-2021-01-17";
```